src/borg/archiver.py

```diff
@@ -1533,7 +1533,9 @@ def do_config(self, args, repository, manifest, key, cache):
             save = cache.cache_config.save
         else:
             config = repository.config
-            save = lambda: repository.save_config(repository.path, repository.config)
+
+            def save():
+                repository.save_config(repository.path, repository.config)
```

Comment:
now as i see it (especially when also considering 1533), i like the lambda version more - for consistency.

so, shall we rather disable the flake8 warning?

Reply:
yeah, I prefer the old one as well...